taipy/common/config/__init__.py

```diff
@@ -55,40 +56,50 @@
-        import os
-
-        if os.environ.get("GENERATING_TAIPY_DOC", None) and os.environ["GENERATING_TAIPY_DOC"] == "true":
-            with (open("config_doc.txt", "a") as f):
-                from inspect import signature
-
-                # Add the documentation for configure methods
-                for exposed_configuration_method, configuration_method in configuration_methods:
-                    annotation = "    @staticmethod\n"
-                    sign = "    def " + exposed_configuration_method + str(signature(configuration_method)) + ":\n"
-                    doc = '        """' + configuration_method.__doc__ + '"""\n'
-                    content = "        pass\n\n"
-                    f.write(annotation + sign + doc + content)
-
-                # Add the documentation for the attribute
-                annotation = '    @property\n'
-                sign = f"    def {attr_name} (self) -> {section.__name__}:\n"
-                if issubclass(section, UniqueSection):
-                    doc = f'        """The configured {section.__name__} section."""\n'
-                elif issubclass(section, Section):
-                    doc = f'        """The configured {section.__name__} sections ."""\n'
-                else:
-                    print(f" ERROR - Invalid section class: {section.__name__}")  # noqa: T201
-                    return
+def __write_func_to_doc(section, attr_name, configuration_methods):
+    if os.environ.get("GENERATING_TAIPY_DOC", None) and os.environ["GENERATING_TAIPY_DOC"] == "true":
+        with open("config_doc.txt", "a") as f:
+            from inspect import signature
+
+            # Add the documentation for configure methods
+            for exposed_configuration_method, configuration_method in configuration_methods:
+                annotation = "    @staticmethod\n"
+                sign = "    def " + exposed_configuration_method + str(signature(configuration_method)) + ":\n"
+                doc = '        """' + configuration_method.__doc__ + '"""\n'
                 content = "        pass\n\n"
                 f.write(annotation + sign + doc + content)
-        return func(section, attr_name, default, configuration_methods, add_to_unconflicted_sections)
+
+            # Add the documentation for the attribute
+            annotation = "    @property\n"
+            sign = f"    def {attr_name} (self) -> {section.__name__}:\n"
+            if issubclass(section, UniqueSection):
+                doc = f'        """The configured {section.__name__} section."""\n'
+            elif issubclass(section, Section):
+                doc = f'        """The configured {section.__name__} sections ."""\n'
+            else:
+                print(f" ERROR - Invalid section class: {section.__name__}")  # noqa: T201
+                return
+            content = "        pass\n\n"
+            f.write(annotation + sign + doc + content)
```

Comment:
This part of the code should only be executed for documenting a section. Not a configure method.